### PR TITLE
Add explanation for answer

### PR DESCRIPTION
**Pull request by Deborah-Digges**

### CODE REVIEW

File: solutions.md [hunk 1]
```diff
@@ -60,9 +60,29 @@ clean :
 > 3. only .dat files recreated
 > 4. only analysis.tar.gz recreated
 
+~~~
```
Comment (mikej888) [inline solutions.md]: I don't think this should be here? The entire answer section is rendering as a code-style block.

Comment (Deborah-Digges) [inline solutions.md]: Oh I'll fix that! Sorry. Should I simply remove these markers or add a different class?

**Comment by Deborah-Digges** [comment]
I've modified the markdown such that only the code is surrounded by code style blocks.
Good catch :-)

File: solutions.md [hunk 2]
```diff
@@ -60,9 +60,29 @@ clean :
 > 3. only .dat files recreated
 > 4. only analysis.tar.gz recreated
 
+~~~ 
+	ANSWER: only analysis.tar.gz recreated.
```
Comment (mikej888) [inline solutions.md]: Replace analysis.tar.gz with `analysis.tar.gz`

File: solutions.md [hunk 3]
```diff
@@ -60,9 +60,29 @@ clean :
 > 3. only .dat files recreated
 > 4. only analysis.tar.gz recreated
 
+~~~ 
+	ANSWER: only analysis.tar.gz recreated. 
+
+    You can check that *.dat is being expanded in the target of the rule for
```
Comment (mikej888) [inline solutions.md]: Replace *.dat with `*.dat`

File: solutions.md [hunk 4]
```diff
@@ -60,9 +60,29 @@ clean :
...
+    analysis.tar.gz by echoing the value of the automatic variable $^
```
Comment (mikej888) [inline solutions.md]: Replace analysis.tar.gz with `analysis.tar.gz` and $^ with `$^`

File: solutions.md [hunk 5]
```diff
@@ -60,9 +60,29 @@ clean :
...
+
+    You can check that *.dat is being expanded in the target of the rule for
+    analysis.tar.gz by echoing the value of the automatic variable $^ 
+    (all dependencies of the current rule).
+
+    analysis.tar.gz: *.dat
+        @echo $^
+        tar -czf $@ $^
+
+    The rules for *.dat are not executed because their corresponding .txt files
```
Comment (mikej888) [inline solutions.md]: Replace *.dat and .txt with `*.dat` and `.txt`

File: solutions.md [hunk 6]
```diff
@@ -60,9 +60,29 @@ clean :
...
+    haven't been modified.
+
+    If you run:
+     $ touch *.dat
+     $ touch books/*.txt
+     $ make analysis.tar.gz
+
+     You will find that the .dat files as well as analysis.tar.gz are recreated.
```
Comment (mikej888) [inline solutions.md]: Replace `*.dat` with `*.dat` and analysis.tar.gz with `analysis.tar.gz`

**Comment by Deborah-Digges** [comment]
Done :-) :+1:

**Comment by mikej888** [comment]
Excellent, thanks Deborah :-)

**Pull request merged**